pkg/helm/handlers/handler_test.go

```diff
@@ -681,25 +686,30 @@ func TestHelmHandlers_Index(t *testing.T) {
 		proxyNewError    error
 		indexFileError   error
 		expectedResponse string
+		urlQuery         string
+		onlyCompatible bool
 	}{
 		{
-			name:           "valid repo index file should return correct response",
-			httpStatusCode: http.StatusOK,
+			name: "valid repo index file should return correct response",
```

Comment:
why this change? we should have another test with filtering set to on.. and this test should remain untouched.

Reply:
Added new test.